website/content/en/preview/troubleshooting.md

```diff
@@ -440,6 +440,13 @@ nodeAffinity:
             values: ['us-east-1a', 'us-east-1b']
 ```
 
+### Log message of `no instance type met the scheduling requirements or had a required offering` is reported
```

Comment:
Can you make this update to all the version of the docs?

Reply:
Sure!